__tests__/app.test.js

```diff
@@ -112,3 +102,51 @@ describe("/api/articles/:article_id", () => {
             })
     })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("GET:200 sends an array of comments corresponding to the requested article_id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toHaveLength(11)
+                expect(comments).toBeSortedBy("created_at", {
+                    descending: true,
+                })
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        body: expect.any(String),
+                        article_id: expect.any(Number),
```

Comment:
are you expecting any `article_id` or are you expecting these comments to belong to a specific article?